docs/reference/installation.rst

```diff
@@ -2,55 +2,69 @@
 e-commerce Bundles installation
 ===============================
 
-2 installation process are available:
-
-* :ref:`quick-install-ref`
-* :ref:`manual-install-ref`
-
-.. _quick-install-ref:
-
-Quick install
-=============
-
-In case you need to work with Sonata e-commerce for a new project or just want to test it for a simple overview, we strongly advise that you use our demo.
-This demo is available `here <http://demo.sonata-project.org>`_ and the code on `Github <https://github.com/sonata-project/sandbox>`_.
-
-
-See the `README.md <https://github.com/sonata-project/sandbox/blob/2.4-develop/README.md>`_ for the dedicated *but very easy* installation process and `CONTRIBUTING.MD <https://github.com/sonata-project/sandbox/blob/2.4-develop/CONTRIBUTING.md>`_  if you want to contribute.
+Prerequisites
+-------------
+
+PHP 7 and Symfony 2.8, >=3.3 or 4 are needed to make this bundle work, there are
+also some Sonata dependencies that need to be installed and configured beforehand:
+
+    - `SonataAdminBundle <https://sonata-project.org/bundles/admin>`_
+    - `SonataEasyExtendsBundle <https://sonata-project.org/bundles/easy-extends>`_
+    - `SonataCoreBundle <https://sonata-project.org/bundles/core>`_
+    - `SonataBlockBundle <https://sonata-project.org/bundles/block>`_
+    - `SonataClassificationBundle <https://sonata-project.org/bundles/classification>`_
+    - `SonataMediaBundle <https://sonata-project.org/bundles/media>`_
+    - `SonataNotificationBundle <https://sonata-project.org/bundles/notification>`_
+    - `SonataDashboardBundle <https://sonata-project.org/bundles/dashboard>`_
+    - `SonataSeoBundle <https://sonata-project.org/bundles/seo>`_
+    - `SonataFormatterBundle <https://sonata-project.org/bundles/formatter>`_
+
+You will need to install those in their 2.0 or 3.0 branches. Follow also
```

Comment:
Maybe leave out the version? as we don't know how will the release cycle go and no one will come and edit this when we release something